app/models/plan.rb

```diff
@@ -450,6 +450,14 @@ def add_user!(user_id, access_type = :commenter)
     end
   end
 
+  ## Update plan identifier.
+  #
+  # Returns Boolean
+  def add_identifier!(identifier)
```

Comment:
I like this being a function on the plan model so we can update it's behavior if we decide to go with some kind of identifier other than the plan's ID

Reply:
Yes! This may be a good spot to place any DOIs if we start minting them for DMPs.